src/main/groovy/io/seqera/wave/service/pairing/socket/PairingWebSocket.groovy

```diff
@@ -48,6 +50,7 @@ import static io.seqera.wave.util.LongRndKey.rndHex
 @CompileStatic
 @Singleton
 @ServerWebSocket("/pairing/{service}/token/{token}{?endpoint}")
+@ExecuteOn(TaskExecutors.IO)
```

Comment:
@munishchouhan, can you please verify if `@ExecuteOn` can also be used with websockets?

Reply:
sure

Reply:
i tried some changes, but still its running on default netty event loop
```
20:59:01.005 [default-nioEventLoopGroup-2-2] DEBUG i.s.w.s.p.socket.PairingWebSocket - Opening pairing session - endpoint: http://localhost:8000/api [sessionId: QPxZ6z5VpHGdjU/9dGVXvg==]
20:59:01.010 [default-nioEventLoopGroup-2-2] DEBUG i.s.w.s.pairing.PairingServiceImpl - Pairing with service 'tower' at address http://localhost:8000/api - pairing id: 149783553036147 (key: 9f9ab8df443d5396dca7dea8cb294ebd)
20:59:31.452 [default-nioEventLoopGroup-2-2] TRACE i.s.w.s.p.socket.PairingWebSocket - Receiving heartbeat - endpoint: http://localhost:8000/api [sessionId: QPxZ6z5VpHGdjU/9dGVXvg==]
21:00:21.444 [default-nioEventLoopGroup-2-2] TRACE i.s.w.s.p.socket.PairingWebSocket - Receiving heartbeat - endpoint: http://localhost:8000/api [sessionId: QPxZ6z5VpHGdjU/9dGVXvg==]
```

Reply:
I have created issue with micronaut team to check about this

Reply:
In the meanwhile, it could be managed manually using a `supplyAsync`

Reply:
ok sure, i will work on that

Reply:
Thinking more it's not needed. The most important one is already async 

https://github.com/seqeralabs/wave/blob/436103b4d6cfd74c67aeab347fb3adcbff4499e2/src/main/groovy/io/seqera/wave/service/pairing/socket/PairingWebSocket.groovy#L97-L108